docs/source/extension.rst

```diff
@@ -294,7 +294,8 @@ close ssh session
 
 - require(module)
 
-Dynamically load modules
+Dynamically load modules loaded in source code or
+installed via npm in node_modules at working directory
```

Comment:
Ok, I'll add a configurable path in config. Working directory is a directory of loaded config yaml.